minoc/src/main/java/org/opencadc/minoc/ArtifactAction.java

```diff
@@ -265,7 +265,13 @@ public void checkReadPermission()
     
     public void checkWritePermission()
         throws AccessControlException, ResourceNotFoundException, TransientException {
-        
+
```

Comment:
In master the check for anon comes before the hack to allow unrestricted write during development. That was on purpose to require authentication but skip authorization. Why has that changed here?

Reply:
I think I copied the hack in front of the auth check for a quick test, then forgot to remove it. Removed.